src/main/asciidoc/jdbc.adoc

```diff
@@ -1,73 +1,71 @@
-
-[[jdbc.introduction]]
-== Introduction
+This chapter points out the specialties for repository support for JDBC. This builds on the core repository support explained in <<repositories>>.
+You should have a sound understanding of the basic concepts explained there.
 
 [[jdbc.why]]
 === Why Spring Data JDBC?
 
-The main persistence API for relational databases in the Java world is certainly JPA, which has it's own Spring Data module.
+The main persistence API for relational databases in the Java world is certainly JPA, which has its own Spring Data module.
 Why is there another one?
 
 JPA does a lot of things in order to help the developer.
-Among others it tracks changes to entities.
+Among other things, it tracks changes to entities.
 It does lazy loading for you.
-It allows to map a wide array of object constructs to an equally wide array of database design.
+It lets you map a wide array of object constructs to an equally wide array of database designs.
 
 This is great and makes a lot of things really easy.
 Just take a look at a basic JPA tutorial.
-But often it gets really confusing why JPA does a certain thing.
-Or things that are really simple conceptually get rather difficult with JPA.
+But it often gets really confusing as to why JPA does a certain thing.
+Also, things that are really simple conceptually get rather difficult with JPA.
 
-Spring Data JDBC aims to be much simpler conceptually:
+Spring Data JDBC aims to be much simpler conceptually, by supporting the following features:
 
-* If you load an entity, SQL statements get executed and once this is done you have a completely loaded entity.
+* If you load an entity, SQL statements get executed.
+Once this is done, you have a completely loaded entity.
 No lazy loading or caching is done.
 
-* If you save and entity it gets saved.
-If you don't it doesn't.
+* If you save an entity, it gets saved.
+If you do not, it does not.
 There is no dirty tracking and no session.
 
-* There is a simplistic model of how to map entities to tables.
+* There is a simple model of how to map entities to tables.
 It probably only works for rather simple cases.
-If you don't like that, just code your strategy yourself.
-Spring Data JDBC will offer only very limited support for customizing the strategy via annotations.
+If you do not like that, you should code your own strategy.
+Spring Data JDBC offers only very limited support for customizing the strategy with annotations.
 
 [[jdbc.domain-driven-design]]
 === Domain Driven Design and Relational Databases.
 
-All Spring Data modules are inspired by the concepts of Repository, Aggregate and Aggregate Root from Domain Driven Design.
-These are possibly even more important for Spring Data JDBC because they are to some extend contrary to normal practice when working with relational databases.
+All Spring Data modules are inspired by the concepts of "`repository`", "`aggregate`", and "`aggregate`" root from Domain Driven Design.
```

Comment:
I don't think "repository", "aggregate" and "aggregate root" should be formatted as code since they are theoretical concepts/ patterns.

And the last one should be "aggregate root" (root being part of the pattern name).

Reply:
The back ticks inside the double quotation marks render as curly quotation marks. They won't show up as code. It's an oddity of Asciidoc. I'm trying to consistently use curly quotation marks in our output.